iwyu_lexer_utils.cc

```diff
@@ -48,14 +56,13 @@ const char* SourceManagerCharacterDataGetter::GetCharacterData(
   return data;
 }
 
-string GetSourceTextUntilEndOfLine(
-    SourceLocation start_loc,
-    const CharacterDataGetterInterface& data_getter) {
+StringRef GetSourceTextUntilEndOfLine(
+    SourceLocation start_loc, const CharacterDataGetterInterface& data_getter) {
```

Comment:
That's fine by me, we might as well follow clang-format's lead when we're touching functions.

Personally, I think the stacked parameter list is nicer, but I bow to the tooling.